Sunday Default.xml

```diff
@@ -1454,6 +1454,386 @@
             <Op id="TransitionPosition" mixEffectBlockIndex="1" position="0"/>
             <Op id="PreviewInput" mixEffectBlockIndex="1" input="Camera20"/>
         </Macro>
+        <Macro index="50" name="VTRW - Play Vid 1" description="">
+            <!-- Make sure Key 3 is off so we can transition it on -->
+            <Op id="KeyOnAir" mixEffectBlockIndex="0" keyIndex="2" onAir="False"/>
+            <!-- Set Animated Lower Third Clip -->
+            <Op id="HyperDeckSetSourceClipIndex" hyperDeckIndex="3" clipIndex="0"/>
+            <Op id="HyperDeckSetSingleClip" hyperDeckIndex="3" singleClipEnabled="True"/>
+            <Op id="HyperDeckSetSpeed" hyperDeckIndex="3" speedPercent="1"/>
+            <!-- Transition: Key 3: On -->
+            <Op id="TransitionStyle" mixEffectBlockIndex="0" style="Mix"/>
```

Comment:
I know it's like this in the original version, but is there a reason we don't just boop the keyer on-air? For these there shouldn't be any need to do a dissolve and it would drastically simplify all these, plus we avoid having to change the transition mode on the director.

Reply:
It's a vestige of when the video didn't fade in, and instead was a static title card that we had to manually fade in. I assume we can guarantee then that all our video title cards in the future will transition themselves in and out? I can try booping it on for 16/17 and have you try it today; if it looks good I'll propagate it to these new ones?

Reply:
That makes sense. Yeah I'm here all day, let me know when you want to try it.